hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/regression/UtilScalePricingCheck.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.services.bdd.suites.regression;
+
+import static com.hedera.services.bdd.spec.HapiSpec.propertyPreservingHapiSpec;
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.getTxnRecord;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.cryptoCreate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.mintToken;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.tokenCreate;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.blockingOrder;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.overridingTwo;
+
+import com.google.protobuf.ByteString;
+import com.hedera.services.bdd.spec.HapiSpec;
+import com.hedera.services.bdd.spec.HapiSpecOperation;
+import com.hedera.services.bdd.suites.HapiSuite;
+import com.hederahashgraph.api.proto.java.TokenType;
+import java.util.List;
+import java.util.function.IntFunction;
+import java.util.stream.IntStream;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class UtilScalePricingCheck extends HapiSuite {
+
+    private static final Logger LOG = LogManager.getLogger(UtilScalePricingCheck.class);
+    private static final String NON_FUNGIBLE_TOKEN = "NON_FUNGIBLE_TOKEN";
+    private static final String MAX_MINTS_PROP = "tokens.nfts.maxAllowedMints";
+    private static final String ENTITY_UTILIZATION_SCALE_FACTOR_PROP = "fees.percentUtilizationScaleFactors";
+
+    public static void main(String... args) {
+        new UtilScalePricingCheck().runSuiteSync();
+    }
+
+    @Override
+    public List<HapiSpec> getSpecsInSuite() {
+        return List.of(nftPriceScalesWithUtilization());
+    }
+
+    private HapiSpec nftPriceScalesWithUtilization() {
+        final var civilian = "civilian";
+        final var maxAllowed = 100;
+        final IntFunction<String> mintOp = i -> "mint" + i;
+        final var standard100ByteMetadata = ByteString.copyFromUtf8(
```

Comment:
Why do we need this?  Is there a charge based on the size of this?  (Asking because I don't know.)

Reply:
Yes, the size of the metadata affects the price of minting an NFT.